.github/workflows/avocado.yml

```diff
@@ -0,0 +1,38 @@
+name: "[TEST-IGNORE] Swagger Avocado"
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  avocado:
+    name: "[TEST-IGNORE] Swagger Avocado"
+
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Must include all branches, for git branch logic in Avocado to work correctly
```

Comment:
For my information what branches other than the current one does Avocado actually look at?

Reply:
Avocado has logic similar to BreakingChanges and LintDiff, where it runs the tool before and after the PR, and attempts to exclude (some) errors if they also occur before the PR.

Longer-term, I think this "feature" should be removed, and all Avocado suppressions should be in suppressions.yaml.  Then, Avocado can run the same both locally and in CI.

Short-term, if we want the check to work unchanged, we need to clone all branches, so Avocado can run `git checkout <pr-base-branch>`.  Unlike other checks, Avocado has the code to checkout and change branches baked right into the tool.